<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wazeingClass</class>
 <widget class="QMainWindow" name="wazeingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>wazeing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CurrentLocation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentLoc">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Status">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Score">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>421</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDescription">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCurrentLoc">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>270</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CurrentLocation:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_radius">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_North">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>North</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_South">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>South</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_East">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>East</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Vest">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Vest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Validate">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>260</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Validate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentLoc_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="wazeing.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>wazeingClass</receiver>
   <slot>slider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_North</sender>
   <signal>clicked()</signal>
   <receiver>wazeingClass</receiver>
   <slot>north()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_South</sender>
   <signal>clicked()</signal>
   <receiver>wazeingClass</receiver>
   <slot>south()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_East</sender>
   <signal>clicked()</signal>
   <receiver>wazeingClass</receiver>
   <slot>east()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>534</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Vest</sender>
   <signal>clicked()</signal>
   <receiver>wazeingClass</receiver>
   <slot>vest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>wazeingClass</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Validate</sender>
   <signal>clicked()</signal>
   <receiver>wazeingClass</receiver>
   <slot>validate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>wazeingClass</receiver>
   <slot>handle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slider()</slot>
  <slot>slot2()</slot>
  <slot>north()</slot>
  <slot>vest()</slot>
  <slot>east()</slot>
  <slot>south()</slot>
  <slot>add()</slot>
  <slot>validate()</slot>
  <slot>handle()</slot>
 </slots>
</ui>
